e2e/tools/validator/src/validator/config/__init__.py

```diff
@@ -32,6 +32,7 @@ class Validator(NamedTuple):
     remote: Remote
     metal: Metal
     prometheus: Prometheus
+    remote_prometheus: Prometheus
```

Comment:
Could you please shed some light into why we need this? 
I would expect the same prometheus on the Baremetal to scrape kepler running on VM.

Reply:
@sthaha this is to support the configuration of having two prometheus instances running in metal and vm. If you configure the same prometheus to scrape metal and vm, you can set the same url for the remote and local prometheus